Updated whitelist definition. (#188)
eProsima/Fast-RTPS-docs: docs/fastdds/transport/whitelist.rst

Interface Whitelist
-------------------

> It is possible to define various network interfaces on the :ref:`transport_tcp_tcp`
> or :ref:`transport_udp_udp` in order to force Fast DDS entities to use exclusively these network interfaces.

Reviewer: ```suggestion
Using *Fast DDS*, it is possible to limit the network interfaces used by :ref:`transport_tcp_tcp` and :ref:`transport_udp_udp`.
```

> This is achieved by adding the IP address corresponding to the interface to the ``interfaceWhiteList``

Reviewer: ```suggestion
This is achieved by adding the interfaces' IP addresses to the ``interfaceWhiteList``
```

field in the :ref:`transport_tcp_transportDescriptor` or :ref:`transport_udp_transportDescriptor`.
Thus, the communication of the entities that define an ``interfaceWhiteList`` is forced through the addresses that

> are defined in it, avoiding the use of the rest of the network interfaces available in the system.

Reviewer: This sentence is bit confusing IMO. The entities do not define a whitelist, since this is defined in the transport descriptor. Furthermore, the transport descriptor is associated with the DomainParticipant, not with any other entities. I agree with the need of restating that if there is a whitelist you only use those interfaces.
